src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperParseTreeRewriter.cs

```diff
@@ -274,7 +340,7 @@ private bool TryRecoverTagHelper(string tagName, RewritingContext context)
 
             foreach (var tag in _tagStack)
```

Comment:
`tag` -> `tracker` since the value isn't the tag, `TagBlock`, or `TagBlockBuilder`